_includes/components/image-example-incorrect.html

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+include
+- image-alt-text
+- image 
+- red-text
```

Comment:
I wonder if we can make this more generic to allow someone to specify if this is the "correct" or "incorrect" variant, and then apply relevant classes based on that.

In particular, I'd think we should avoid the "red" implementation detail and stick to semantics, where here we might call it something like `do_dont_text` ?

Reply:
Renamed. I'm struggling to make the semantic naming stick in my head, working on it.